internal/proxy/task_search.go

```diff
@@ -328,6 +327,8 @@ func (t *searchTask) PreExecute(ctx context.Context) error {
 			return err
 		}
 
+		t.request.OutputFields = append(t.request.OutputFields, common.TimeStampFieldName)
+		outputFieldIDs = append(outputFieldIDs, common.TimeStampField)
 		t.SearchRequest.OutputFieldsId = outputFieldIDs
```

Comment:
The user doesn't need the timestamp filed as output field, please remove this field at proxy

Reply:
ok,I will delete it at proxy